src/py/flwr/cli/app.py

```diff
@@ -14,7 +14,7 @@
 # ==============================================================================
 """Flower command line interface."""
 
-
+from build import __version__
```

Comment:
Instead of pulling `__version__` from build, I would suggest to use `package_version` from `flwr.common.version`: 
`from flwr.common.version import package_version`